### PR TITLE
Add call for paper section

### PR DESCRIPTION
**Pull request by imeugenia**
cfp

### CODE REVIEW

File: themes/falconf/layouts/index.html [hunk 1]
```diff
@@ -7,6 +7,9 @@
       {{ if .Site.Params.sectionTickets.enabled }}
         <a href="#tickets">Tickets</a>
       {{ end }}
+      {{ if .Site.Params.sectionCallForPaper.enabled }}
```
Comment (imeugenia) [inline themes/falconf/layouts/index.html]: Should it be here?

Comment (skibish) [reply themes/falconf/layouts/index.html]: yes

File: themes/falconf/layouts/index.html [hunk 2]
```diff
@@ -101,6 +104,16 @@ <h1>Tickets</h1>
           </div>
         </div>
         {{ end }}
+        <!-- Section: Call for paper -->
```
Comment (skibish) [inline themes/falconf/layouts/index.html]: Also add `if` bock here for section

File: themes/falconf/layouts/index.html (hunk 1 above)
Comment (skibish) [reply themes/falconf/layouts/index.html]: And in side menu this also is present?

File: themes/falconf/layouts/index.html (hunk 2 above)
Comment (imeugenia) [reply themes/falconf/layouts/index.html]: Done!

**Comment by skibish** [comment]
There is some conflict, after that you can squash & merge

**Pull request merged**